src/ducks/balance/Balance.jsx

```diff
@@ -137,17 +136,6 @@ const BalanceRow = connect(state => ({
   filteringDoc: getFilteringDoc(state)
 }))(_BalanceRow)
 
-BalanceRow.propTypes = {
-  accountOrGroup: or([
-    {
-      account: PropTypes.object.isRequired
-    },
-    {
-      group: PropTypes.object.isRequired
-    }
-  ]).isRequired
-}
-
```

Comment:
So the BalanceRow has no propTypes ?

Reply:
I create a custom validator https://github.com/cozy/cozy-banks/pull/574/files#diff-e7d9783b37787e694e296c56a99fba5cR140 I remove `airbnb-prop-types` dependency ;)